text/0000-grid-inventory.md

```diff
@@ -0,0 +1,126 @@
+information from external systems of record and recording the results in state.
+
+Grid Inventory also aims to serve as a foundation for future potential Grid
+capabilities that may require inventory related information, such as purchasing
+and logistics.
+
+# Motivation
+[motivation]: #motivation
+
+The Grid Inventory implementation is designed to enable shared visibility of
+summary inventory information for any defined Grid Product and Grid Location
+between participants.  Management of inventory levels is a common concept
+within supply chain solutions and is a key primary business activity related
+to all purchases of physical goods from suppliers, as well as all sales of
+physical goods to customers.  Management of inventory levels is typically
+coordinated very manually and very independently between parties across siloed
+systems and processes.  This siloed and manual coordination of inventory
+between parties can lead to substantial pain-points, including:
+inefficiencies, waste, and poor coordination of inventory levels across
+the supply chain.
+
+This Grid Inventory concept aims to solve these issues by offering a common
+industry solution for vertically integrating inventory information between
+parties existing systems of record.
+
+The Grid Inventory design addresses several use cases, including:
+
+- View Current Inventory Status Data: Viewing current summary inventory
+  quantities and inventory statuses between trading partners for defined Grid
+  Products and Grid Locations.
+
+- View Historical Inventory Activity Data: Viewing historical inventory
+  activity between trading partners for defined Grid Products and Grid
+  Locations.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+TBD - based on discussion.  We are exploring designs based on:
+
+ - GS1’s EDI Inventory Report - Business Message Standard.
+ - GS1's X12 EDI Standards.
+ - GS1's EPCIS Standard.
+ - Other standards as relevant.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+TBD - based on discussion.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+By leveraging GS1’s EDI Inventory Report standards, only a summary view of
+inventory information can be visible between parties.  Visibility of batch/lot
+or serialized product instance information will not be visible in inventory
+using this initial Grid Inventory design.  In addition, in order for this
+solution to offer maximum value, it will likely require behavioral changes by
+end-users who are transacting in existing systems to perform inventory stock
+adjustment or movement transactions more real-time and more frequently.  Often,
+these production reporting transactions are done not in real-time, for example
+at the end of each shift, week, or even in some cases at month-end.
+More frequent, or ideally real-time submission of transactions can help ensure
+the physical and digital versions of the supply chain remain in sync.
```

Comment:
Is there something about Grid Inventory design that prevents "more frequent" updates?  We should expand on that if so.  If this is really just saying "a lot of companies aren't setup to do frequent inventory updates" that's maybe not a drawback to this proposal?

Reply:
As I read the PR content and try to distill the scope and balance of real-time availability of actual inventory on-hand vs. the available to promise (ATP) based on my upstream suppliers (as the supplier, and I ask my suppliers who ask more suppliers), I'm wondering what the spec in this RFC would say about including a duration and effort to help address the questions of not necessarily "what do you have?" but "I need X by Y at price Z".

Reply:
Thank you Dave & desktophero for these comments.    

@davececchi  - Regarding more frequent updates, you are correct, it is not a limitation to the design of Grid Inventory, more-so how companies could provide updates via external systems and the current typical behaviors of transaction entry into those external systems.  Given that this is a local concern and not directly part of Grid inventory, we can omit this statement from the design.  However, i do think it is important from an awareness perspective when thinking about how companies might implement Grid Inventory via integration to their existing production reporting and inventory systems of record.

@desktophero  - This is an interesting point regarding "i need X by Y at price Z".   I could see this being an interesting intersection with something like a Grid Purchase Order component.